src/citrine/informatics/predictors/mean_property_predictor.py

```diff
@@ -22,11 +30,11 @@ class MeanPropertyPredictor(
         description of the predictor
     input_descriptor: FormulationDescriptor
         descriptor that represents the input formulation
-    properties: List[RealDescriptor]
-        List of descriptors to featurize
-    p: int
-        Power of the `generalized mean <https://en.wikipedia.org/wiki/Generalized_mean>`_.
-        Only integer powers are supported.
+    properties: List[Union[RealDescriptor, CategoricalDescriptor]]
+        list of real or categorical descriptors to featurize
```

Comment:
```suggestion
        List of real or categorical descriptors to featurize
```
Match capitalization pattern